plugin.video.fox.news/addon.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.fox.news"
+       name="Fox News"
+       version="6.0.1"
+       provider-name="t1m">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.t1mlib" version="1.0.18"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource"
+            library="default.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Fox News Videos</summary>
+    <description lang="en_GB">Fox News Videos</description>
+    <disclaimer lang="en_GB">Feel free to use this script. For information visit the wiki.</disclaimer>
+    <platform>all</platform>
+    <language>en</language>
+    <license>LGPL-2.0-or-later</license>
```

Comment:
I don't think the license file you are adding is LGPL, please confirm

Reply:
Not sure, isn't it? 
https://spdx.org/licenses/LGPL-2.0-or-later.html

Is there a difference between a GNU Public License and a GNU Library public license?
I

Reply:
NM. I found the right one.
https://spdx.org/licenses/GPL-2.0-or-later.html

Reply:
done